src/Tribe/Ticket_Object.php

```diff
@@ -706,6 +709,10 @@ public function inventory() {
 			}
 
 			$inventory = min( $inventory );
+
+			// Prevent inventory higher than the stock.
+			$inventory = min( $inventory, $this->stock() );
```

Comment:
Can't we just update the previous line? We are already doing `min` before this line.